viite-UI/src/view/ProjectEditForm.js

```diff
@@ -103,6 +103,7 @@
         '</header>' +
         '<div class="wrapper read-only">' +
         '<div class="form form-horizontal form-dark">' +
+        '<label style="color: #fff">JATKA VALITSEMALLA KOHDE KARTALTA.</label>' +
```

Comment:
Would be better to define this style in css:
```
.form-dark label {
  color: #fff;
}

```
`'<label>JATKA VALITSEMALLA KOHDE KARTALTA.</label>' +`